docs/infrastructure.md

```diff
@@ -0,0 +1,87 @@
+## Infrastructure
+
+### [General Deployment Process Overview Doc](https://github.com/broadinstitute/dsp-devops-wiki/wiki/deployment_process)
+
+### Infrastructure Overview
+The Google Infrastructure is managed via Terraform and nearly all the kubenetes deployments and such are done via a Helm Chart. See [General Deployment Process Overview Doc](https://github.com/broadinstitute/dsp-devops-wiki/wiki/deployment_process) for more in depth guide. Datarepo is deployed on Kubernetes, Google Cloud Sql (postgres) and contains the following containers:
```

Comment:
Just my thought, take it or leave it!

What I found particularly helpful in the talk last week was how you clearly distinguished between what Terraform handles vs. what helm handles. I know this is mostly covered in the linked page, but maybe expanding on that a bit here would be helpful? 
In particular, this is what I found useful (also, please correct if not right):
> Terraform provisions the underlying infrastructure within the google project, which handles everything up to getting the Kubernetes node pool set up. Helm handles everything inside of the Kubernetes node, including nearly all of the Kubernetes deployments.

Reply:
Personally would rather reference the other doc vs copy pasta.